soh/soh/Enhancements/debugger/debugSaveEditor.cpp

```diff
@@ -268,6 +284,14 @@ void DrawInfoTab() {
 
     ImGui::Text("Name: %s", name.c_str());
     InsertHelpHoverText("Player Name");
+    std::string nameID;
```

Comment:
If `ImGui::InputScalar` can take a `char` I'd change this to that.

Reply:
Unfortunately, `ImGui::InputScalar` only works with numerical types. I don't think there'd be a way to get a character in there.
I've also tested using `ImGui::TextEntry`, but 1) the conversion from ASCII to the character points that OoT actually reads is a PITA, and 2) I'm not able to edit the entire array without somehow getting into a buffer overflow through this method.